index.bs

```diff
@@ -1224,54 +1229,62 @@ runs these steps:
          1. [=Assert=]: |data| is a {{USVString}}.
          1. Let |dataBytes| be the result of [=UTF-8 encoding=] |data|.
       1. If |writePosition| is larger than |oldSize|,
-         append |writePosition| - |oldSize| 0x00 (NUL) bytes to the end of |stream|.[=[[buffer]]=].
+         append |writePosition| - |oldSize| 0x00 (NUL) bytes to the end of
+         |stream|'s [=[[buffer]]=].
 
          Note: Implementations are expected to behave as if the skipped over file contents
          are indeed filled with NUL bytes. That doesn't mean these bytes have to actually be
          written to disk and take up disk space. Instead most file systems support so called
          sparse files, where these NUL bytes don't take up actual disk space.
 
-      1. Let |head| be a [=byte sequence=] containing the first |writePosition| bytes of |stream|.[=[[buffer]]=].
+      1. Let |head| be a [=byte sequence=] containing the first |writePosition|
+         bytes of |stream|'s[=[[buffer]]=].
       1. Let |tail| be an empty [=byte sequence=].
-      1. If |writePosition| + |data|.[=byte sequence/length=] is smaller than |oldSize|:
+      1. If |writePosition| + |data|'s [=byte sequence/length=] is smaller than |oldSize|:
          1. Let |tail| be a [=byte sequence=] containing the last
-            |oldSize| - (|writePosition| + |data|.[=byte sequence/length=]) bytes of |stream|.[=[[buffer]]=].
-      1. Set |stream|.[=[[buffer]]=] to the concatenation of |head|, |data| and |tail|.
-      1. If the operations modifying |stream|.[=[[buffer]]=] in the previous steps failed
-         due to exceeding the [=storage quota=], [=/reject=] |p| with a
-         "{{QuotaExceededError}}" {{DOMException}} and abort these steps,
-         leaving |stream|.[=[[buffer]]=] unmodified.
+            |oldSize| - (|writePosition| + |data|'s [=byte sequence/length=])
+            bytes of |stream|'s [=[[buffer]]=].
+      1. Set |stream|'s [=[[buffer]]=] to the concatenation of
+         |head|, |data| and |tail|.
+      1. If the operations modifying |stream|'s [=[[buffer]]=] in the
+         previous steps failed due to exceeding the [=storage quota=],
+         [=/reject=] |p| with a "{{QuotaExceededError}}" {{DOMException}}
+         and abort these steps, leaving |stream|'s [=[[buffer]]=] unmodified.
 
          Note: [=Storage quota=] only applies to files stored in the [=origin private file system=].
          However this operation could still fail for other files, for example if the disk being written
          to runs out of disk space.
-      1. Set |stream|.[=[[seekOffset]]=] to |writePosition| + |data|.[=byte sequence/length=].
+      1. Set |stream|'s [=[[seekOffset]]=] to |writePosition| + |data|'s [=byte sequence/length=].
       1. [=/Resolve=] |p|.
     1. Otherwise, if |command| is {{WriteCommandType/"seek"}}:
-      1. If |chunk|.{{WriteParams/position}} is `undefined`,
+      1. [=Assert=]: |chunk| is a [=/dictionary=].
```

Comment:
not sure whether this counts as a non-editorial change... the previous text assumes this to be true